src/components/UserSelector.tsx

```diff
@@ -72,11 +68,9 @@ export const UserSelector: React.FC<Props> = ({
             <a
               key={user.id}
               href={`#user-${user.id}`}
-              onClick={() => {
-                onChange(user);
-              }}
+              onClick={() => selectAuthor(user)}
               className={classNames('dropdown-item', {
-                'is-active': user.id === selectedUser?.id,
+                'is-active': user.id === author?.id,
```

Comment:
This line checks if the current user is the selected author. Ensure that `author` is always defined or handle the case where `author` might be undefined to avoid potential runtime errors.